### DISCUSSION TITLE
Welcome to Rewrite Discussions!

### DISCUSSION THREAD

**Discussion by jkschneider** [openrewrite/rewrite#209] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jbduncan** [comment]
@jkschneider Thanks a lot for starting up a public Slack for this project! I'm interested in joining, but it seems to ask me for a moderne.io email address. Am I missing something here?

**Reply by aegershman** [reply]
@jbduncan Hm. Yes I'm seeing that too. I think the link in @jkschneider 's discussion body might not be the same invite link, maybe. Could you try the ["Join us on Slack"](https://docs.openrewrite.org/) link at the top of the https://docs.openrewrite.org page? (the same link is in the README body of the repo here: https://github.com/openrewrite/rewrite#semantic-code-search-and-transformation)

**Reply by jbduncan** [reply]
@aegershman Thank you very much for getting back to me.

When I click on the link from the README to you linked to, it also asks me for a moderne.io email address. But clicking on the "Join us on Slack" link in the docs doesn't. I was able to sign up there just now, so thanks for pointing me in the right direction!

To summarise, it looks to me that the Slack links in the OP of this discussion and on the README are invalid.

**Comment by timtebeek** [comment]
Closing as stale; feel free to open a new discussion if this is still relevant.
